src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -5798,14 +5867,14 @@ void CmdSketcherConstrainPerpendicular::activated(int iMsg)
 
             const Part::Geometry* geom2 = Obj->getGeometry(GeoId2);
 
-            if (geom2 && isBSplineCurve(*geom2)) {
-                // unsupported until normal to B-spline at any point implemented.
-                Gui::TranslatedUserWarning(
-                    Obj,
-                    QObject::tr("Wrong selection"),
-                    QObject::tr("Perpendicular to B-spline edge currently unsupported."));
-                return;
-            }
+            // if (geom2 && isBSplineCurve(*geom2)) {
```

Comment:
Can we delete instead of commenting? git will remember it for us if we need it again (here and elsewhere the same technique appears).